examples/enable-operator-and-auto-instrumentation/README.md

```diff
@@ -0,0 +1,189 @@
+# Example of chart configuration
+
+## How to deploy the OpenTelemetry Operator and Java auto-instrumentation
+> :Notice: Operator related features should be considered to have an alpha
+> maturity level and be experimental. There may be breaking changes or
+> Operator features may be replaced entirely with a better alternative
+> solution in the future.
+
+In the following example we will show how to instrument
+[spring-petclinic-microservices](https://github.com/spring-petclinic/spring-petclinic-microservices).
+
+## Getting started
+### 1. If the common [cert-manager](https://cert-manager.io/docs/installation/kubectl/) is not already deployed and available in the cluster, then deploy it.
+
+If the cert-manager was already installed, you'd likely find it in the
+cert-manager namespace.'
+
+```
+kubectl get pods -n cert-manager
+# NAME                                       READY   STATUS    RESTARTS   AGE
+# cert-manager-6b466d8fb4-njt5n              1/1     Running   0          5h26m```
+```
+
+If it does need to be installed, run this.
+
+```
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.2/cert-manager.yaml
+```
+
+The cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
```

Comment:
I'd move this to be the first paragraph in this section and ideally explain why it's needed for the example.

Reply:
Done